.template/variants/web/spec/support/webpack.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.configure do |config|
-  config.before(:suite) do
+  config.when_first_matching_example_defined(type: :system) do
```

Comment:
@hoangmirs Do you think we can add the `when_first_matching_example_defined` to the `/spec/support/assets.rb` file too 🤔